### PR TITLE
APPINT-32811: CVE for commons-codec-1.6 on studio 7.3

### PR DESCRIPTION
**Pull request by vdrokov**

### CODE REVIEW

**Review by apoltavtsev — APPROVED** [review]
OK

File: main/plugins/org.talend.designer.esb.components.ws.consumer/plugin.xml [hunk 1]
```diff
@@ -407,7 +407,7 @@
             bundleID="" />
         <libraryNeeded context="plugin:org.apache.commons.codec"
             name="org.apache.commons.codec_1.6.0.v201305230611.jar"
-            id="commons-codec-1.6"
+            id="commons-codec-1.15"
             uripath="platform:/base/plugins/org.apache.commons.codec_1.6.0.v201305230611.jar"
```
Comment (xldai) [inline main/plugins/org.talend.designer.esb.components.ws.consumer/plugin.xml:411]: org.apache.commons.codec_1.6.0.v201305230611.jar still?

**Review by xldai — CHANGES_REQUESTED** [review]

File: main/plugins/org.talend.designer.camel.components.localprovider/components/cMessagingEndpoint/cMessagingEndpoint_java.xml [hunk 2]
```diff
@@ -317,7 +317,7 @@
             <IMPORT MODULE_GROUP="camel-jsonpath-alldep" REQUIRED_IF="HOTLIBS.LIBPATH IN ['JSONPATH']" />
             <IMPORT MODULE_GROUP="camel-saxon-alldep" REQUIRED_IF="HOTLIBS.LIBPATH IN ['SAXON']" />
             <!-- additional deps which excluded by camel-shape build -->
-            <IMPORT MODULE="commons-codec-1.6" MVN="mvn:commons-codec/commons-codec/1.6" REQUIRED="true" />
+            <IMPORT MODULE="commons-codec-1.15" MVN="mvn:commons-codec/commons-codec/1.6" REQUIRED="true" />
```
Comment (xldai) [inline main/plugins/org.talend.designer.camel.components.localprovider/components/cMessagingEndpoint/cMessagingEndpoint_java.xml:320]: MVN url still 1.6

**Review by xldai — CHANGES_REQUESTED** [review]

**Review by apoltavtsev — APPROVED** [review]
LGTM

**Review by csuntalendbj — APPROVED** [review]
LGTM

**Review by xldai — APPROVED** [review]
Corrected the fixes.

**Pull request merged**